tests/tasks/test_services.py

```diff
@@ -0,0 +1,195 @@
+import pytest
+from django.contrib.contenttypes.models import ContentType
+from freezegun import freeze_time
+
+from vitrina.orgs.factories import OrganizationFactory, RepresentativeFactory
+from vitrina.orgs.models import Representative
+from vitrina.tasks.factories import TaskFactory
+from vitrina.tasks.services import get_active_tasks
+from vitrina.users.factories import UserFactory
+
+
+@pytest.mark.django_db
+def test_get_active_tasks():
+    organization = OrganizationFactory()
+    user = UserFactory(organization=organization)
+    RepresentativeFactory(
+        content_type=ContentType.objects.get_for_model(organization),
+        object_id=organization.pk,
+        user=user
+    )
+
+    task_for_user = TaskFactory(user=user)
+    task_for_organization1 = TaskFactory(organization=organization)
+    task_for_organization2 = TaskFactory(organization=OrganizationFactory())
+
+    active_tasks = get_active_tasks(user)
+    assert list(active_tasks) == [
+        task_for_user,
+        task_for_organization1,
+    ]
+
+
+@pytest.mark.django_db
+def test_get_active_tasks_with_task_organization_supervisor_after_5_days():
+    with freeze_time("2022-11-18"):
+        parent_organization = OrganizationFactory()
+        child_organization = parent_organization.add_child(instance=OrganizationFactory.build())
+
+        task_for_parent_organization = TaskFactory(organization=parent_organization)
+        task_for_child_organization = TaskFactory(organization=child_organization)
+
+        parent_organization_user = UserFactory(organization=parent_organization)
+        child_organization_user = UserFactory(organization=child_organization)
+
+        RepresentativeFactory(
+            content_type=ContentType.objects.get_for_model(parent_organization),
+            object_id=parent_organization.pk,
+            role=Representative.COORDINATOR,
+            user=parent_organization_user
+        )
+        RepresentativeFactory(
+            content_type=ContentType.objects.get_for_model(child_organization),
+            object_id=child_organization.pk,
+            role=Representative.COORDINATOR,
+            user=child_organization_user
+        )
+
+    with freeze_time("2022-11-23"):
```

Comment:
I would avoid using freezegun, and instead pass `now` argument to `get_active_tasks`, if this argument is not given, then `datetime.date.today()` would be assumed. freezegun is too aggressive and mocks everything related to dates and some times this results in an unexpected and hard to debug bugs.

If it is possible to solve this with a now argument, it is always a better idea.